jsparagus/emit/rust.py

```diff
@@ -468,9 +471,9 @@ def write_action(indent, act, is_packed):
             assert not act.is_inconsistent()
             if isinstance(act, Reduce):
                 value = "value"
-                try:
+                if value in is_packed:
                     packed = is_packed[value]
-                except:
+                else:
                     packed = False
```

Comment:
For your information I adopted the first style as the code is supposed to be faster as there is only a single dictionary lookup. Also, I do not think this matter much.

Reply:
Bare `except` is usually wrong. It catches programmer errors like `NameError` and even Ctrl+C (implemented in Python as a `KeyboardInterrupt` exception) and `sys.exit()` (`SystemExit`).

`except KeyError` would be fine, or what I wrote; certainly the performance is irrelevant in this particular case...